### PR TITLE
Powerelectronics

### PR DESCRIPTION
**Pull request by olfline**
Added some blocks, which are useful for threephase networks:
 - three line wire
 - AC/DC converter (single phase and threephase)
 - DC/AC converter (single phase and threephase)
 - grid node

Regarding the manual:
 - the three line wire was added to the multiwire section
 - the blocks were added to the blocks section
 - An example for the blocks in action at the example section

thanks to the creators for the easy reusable shapes!!

### CODE REVIEW

File: tex/pgfcircbipoles.tex [hunk 1]
```diff
@@ -255,7 +255,7 @@
 {bmultiwire}
 {\ctikzvalof{bipoles/multiwire/height}}
 {\ctikzvalof{bipoles/multiwire/width}}
-{
+{blocks
```
Comment (Rmano) [inline tex/pgfcircbipoles.tex]: This is strange --- I think it's a typo? (BTW, thanks for the PR; will review as soon as I can).

Comment (olfline) [reply tex/pgfcircbipoles.tex]: I cannot explain this one as did mostly copy paste. 
deleted it anyways, thx!

Comment (Rmano) [reply tex/pgfcircbipoles.tex]: Ok!

**Comment by Rmano** [comment]
Ok, nice. I like it; I will merge. Just a couple of comments:
1 - could you add this patch to CHANGELOG? I do not know how and if I can add a commit to this PR...

```
@@ -4,6 +4,7 @@ The major changes among the different circuitikz versions are listed here. See <
 * Version 1.1.2 (unreleased)
 
     - bumped version number
+    - Blocks and component for three-phase networks (3-lines wire, AC/DC and DC/AC converters blocks and grid node block) added by user `@olfline` on GitHub.
 
 * Version 1.1.1 (2020-04-24)
```

...and feel free to change the attribution. Also, if you want, you can add some attribution in the manual, probably near the blocks. We usually use this kind of notation:

```latex
\begin{groupdesc}
  [...]
    \circuitdesc*{pground}{Protective ground}{}
    \circuitdesc{cground}{Chassis ground\footnotemark}{}
    \footnotetext{These last three were contributed by Luigi «Liverpool»}
    \circuitdesc{eground}{European style ground}{}
    \circuitdesc{eground2}{European style ground, version 2\footnotemark}{}
    \footnotetext{These last two were contributed by \texttt{@fotesan}}
\end{groupdesc}
```latex

**Pull request merged**